app/assets/javascripts/accounts/new/basic_teacher_info.jsx

```diff
@@ -62,10 +62,9 @@ EC.BasicTeacherInfo = React.createClass({
                     </div>
                     <div className='row'>
                         <div className='col-xs-8'>
-                            <div className='text-align-center'>By signing up, you agree to our
-                                <a href='/tos'>terms of service</a>
-                                and
-                                <a href='/privacy'>privacy policy</a>.</div>
+                            <div className='text-align-center'>
+                              By signing up, you agree to our <a href='/tos'>terms of service</a> and <a href='/privacy'>privacy policy</a>.
```

Comment:
@jmsardina -- I agree. Feel free to update the PR with that -- we'll have it all merged in this week.